packages/apps-config/src/settings/ethereumChains.ts

```diff
@@ -8,5 +8,5 @@ export const ethereumChains = [
   'moonbase-alphanet',
   'moonbeam-alphanet',
   'moonbeam-standalone',
-  'moonbeam'
```

Comment:
Not sure we want to drop `moonbeam` here? (I guess it is still valid for some chains?)

Reply:
Thank you